pages/tools/slack/emoji.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Slack emoji usage and best practices
+questions:
+  - admins-slack
+cSpell: 
```

Comment:
This is a new tag to me, but making sure it's supposed to be left blank

Reply:
New spellchecker feature - I add the code in case in case it flags something erroneously, but it's ok if it's blank.